src/CLASS2/pair_lj_class2.cpp

```diff
@@ -35,7 +35,7 @@ PairLJClass2::PairLJClass2(LAMMPS *lmp) : Pair(lmp)
 {
   respa_enable = 1;
   writedata = 1;
-  centroidstressflag = 1;
+  centroidstressflag = CENTROID_SAME;
```

Comment:
this is the default - the point of changing the default was to remove this lines from all the true pairwise pair styles